cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,106 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.5.1 <http://docs.cask.co/cdap/3.5.1/index.html>`__
+=============================================================
+
+Known Issues
+------------
+
+- :cask-issue:`CDAP-7175` - Though an underlying issue has been fixed in the upgrade tool
+  to allow it to run on a CDAP instance with authorization enabled, **if you are upgrading
+  an authorization-enabled CDAP instance,** you will need to give the *cdap* user *ADMIN*
+  privileges on all existing CDAP namespaces. **Note:** the *ADMIN* privilege does not
+  give the *cdap* user *READ* or *WRITE* privileges on the namespaces. As these are
+  temporary grants required only for the purpose of running the upgrade tool, once the
+  upgrade is complete, these grants can be revoked. (Alternatively, you can disable
+  authorization before running the upgrade tool and then re-enable it after the upgrade is
+  completed. See the :ref:`Administration Manual: Authorization <admin-authorization>` for
+  details.)
+
+- :cask-issue:`CDAP-7179` - In CDAP 3.5.0, new ``kafka.server.*`` properties replace older
+  properties such as ``kafka.log.dir``. Though the older properties were marked as
+  deprecated, they are in fact being ignored, and need to be replaced with the current
```

Comment:
according to the bug, this is actually not true.  old properties in cdap-site.xml are not ignored.  The issue only comes into play when using CM, because the CSD might generate a cdap-site.xml with both old and new property names, in which the new property names win.  and this only pertains to the kafka.log.dir (vs kafka.server.log.dirs) property, since the CSD autogenerates a default for it using the new property name.